_component_design/choice-tiles.md

```diff
@@ -4,42 +4,65 @@ parent: Inputs and Controls
 layout: component
 category: Components
 usage: >
-  Choice tiles are an alternative selector pattern to radio buttons and checkboxes that provide a means of both displaying choices more visually and of providing a more detailed description of a choice.
+  A choice tile represents a selection that a user can make on a page. As an alternative to a radio button or checkbox, a choice tile contains an icon that graphically represents the choice and a description that provides more detail about the choice.
 preview-image: preview-images/choice-tile.svg
 status: stable
 resource: true
-last-modified: 2017-11-12
-helix-ui-css: true
-helix-ui-javascript: true
-pagelink: https://rackerlabs.github.io/helix-ui/components/choice-tiles/
+last-modified: 2019-01-03
+helix-ui-css: false
+helix-ui-javascript: false
 ---
 
 {% include toc.html %}
 
 <section class="static-section" markdown="1">
 
-## Introduction
-
 <div class="hxRow" markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use choice tiles when you want to use iconography to present a set of choices to the user.
```

Comment:
Use when you want to increase the selectable space, a larger selectable frame and easier to make a selection.  A choice tile focuses the attention of the user...and allows you to add more context as you can have a name, icon, and description. richer information experience. much bigger click space...richer, in context information. makes choices clearer and its easier to interact with when considering mobile.

Reply:
Done.